packages/core/src/utils/mediatorhelpers.ts

```diff
@@ -1,16 +1,20 @@
 import { Agent, MediationRecord } from '@credo-ts/core'
 
 export const isMediatorInvitation = async (agent: Agent, url: string): Promise<boolean> => {
-  const invitation = await agent.oob.parseInvitation(url)
-  if (!invitation) {
+  try {
+    const invitation = await agent.oob.parseInvitation(url)
+    if (!invitation) {
```

Comment:
I might throw in a few spaces between the if conditionals. It has no effect other than to make the code less dense and, maybe arguably, easier to read. Plus spaces are free :)